TCMSApp/src/client/index.html

```diff
@@ -103,6 +106,8 @@
     <script src="/src/client/app/widgets/ht-widget-header.directive.js"></script>
     <!-- endinject -->
 
+    <script src="http://code.jquery.com/jquery-1.7.1.min.js"></script>
```

Comment:
we use constant service for out vendor libraries. can we use it here too? (I saw @ValeriiMal  managed to implement that)